bciers/libs/e2e/src/utils/constants.ts

```diff
@@ -95,3 +95,8 @@ export const appRouteRoles: Record<AppRoute, UserRole[]> = {
   ],
   [AppRoute.USERS]: [UserRole.CAS_ADMIN, UserRole.INDUSTRY_USER_ADMIN],
 };
+
+export const DJANGO_API_BASE_URL = "http://127.0.0.1:8000/api";
```

Comment:
Since `DJANGO_API_BASE_URL` doesn’t change, can we include it in the `COMPLIANCE_E2E_INTEGRATION_STUB_PATH` constant? That way, we won’t need to import it in other modules.
Another option would be to use `API_URL` from `.env` file.